lib/index.js

```diff
@@ -24,10 +24,10 @@ class Oppsy extends Events.EventEmitter {
           psup: ProcessMonitor.uptime,
           psmem: ProcessMonitor.memoryUsage,
           psdelay: ProcessMonitor.delay,
-          requests: this._networkMonitor.requests,
-          concurrents: this._networkMonitor.concurrents,
-          responseTimes: this._networkMonitor.responseTimes,
-          sockets: this._networkMonitor.sockets
+          requests: this._networkMonitor.requests.bind(this._networkMonitor),
```

Comment:
Or you can drop the prototypes entirely and just attach them to `this` in the constructor. That's probably a cleaner solution.